### PR TITLE
Resolved conflict

### PR DESCRIPTION
**Pull request by Talsk14**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,34 @@
...
+                <td>Middle FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+
+            <tr>
+                <td>2019-present</td>
+                <td>Senior FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+        </table>
+            
+            <!-- <li><p>Junior FullStack Developer at <a href="https://careers.epam.ua/">EPAM</a></p></li>
```
Comment (RomanIlkiv) [inline index.html:35]: remove commented code

File: index.html [hunk 2]
```diff
@@ -7,6 +7,34 @@
...
+            </tr>
+            <tr>
+                <td>2015-2016</td>
+                <td>Junior FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+            <tr>
+                <td>2016-2019</td>
+                <td>Middle FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+
```
Comment (RomanIlkiv) [inline index.html:28]: redundant space

File: index.html [hunk 3]
```diff
@@ -7,6 +7,34 @@
...
+                <th>Period</th>
+                <th>Position</th>
+            </tr>
+            <tr>
+                <td>2015-2016</td>
+                <td>Junior FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+            <tr>
+                <td>2016-2019</td>
+                <td>Middle FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
```
Comment (RomanIlkiv) [inline index.html:26]: do not add spaces between =

should be like this: _href="https://humbleteam.com/"_

File: index.html [hunk 4]
```diff
@@ -7,6 +7,34 @@
...
+        <h3>Senior FullStack Developer</h3>
+        <p><u><strong>Work Experience</strong></u></p>
+        <table border="1">
+            <tr>
+                <th>Period</th>
+                <th>Position</th>
+            </tr>
+            <tr>
+                <td>2015-2016</td>
+                <td>Junior FullStack Developer at <a href = "https://humbleteam.com/">Humbleteam</a></td>
```
Comment (RomanIlkiv) [inline index.html:22]: the same here

File: index.html [hunk 5]
```diff
@@ -7,6 +7,32 @@
...
+                <td>2016-2019</td>
+                <td>Middle FullStack Developer at <a href="https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+
+            <tr>
+                <td>2019-present</td>
+                <td>Senior FullStack Developer at <a href="https://humbleteam.com/">Humbleteam</a></td>
+            </tr>
+        </table>
+
```
Comment (RomanIlkiv) [inline index.html:34]: redundant spaces

**Review by RomanIlkiv — APPROVED** [review]

**Comment by RomanIlkiv** [comment]
you have to resolve conflicts in your PR
the same like we did previous time

**Comment by Talsk14** [comment]
> you have to resolve conflicts in your PR the same like we did previous time

I've resolved this conflict

**Pull request merged**